pkg/client/didexchange/client.go

```diff
@@ -133,11 +136,17 @@ func (c *Client) QueryConnections(request *QueryConnectionsParams) ([]*QueryConn
 	}, nil
 }
 
-// QueryConnectionByID fetches single connection record for given id
-func (c *Client) QueryConnectionByID(id string) (*QueryConnectionResult, error) {
-	// TODO sample response, to be implemented as part of #226
+// GetConnection fetches single connection record for given id
+func (c *Client) GetConnection(connectionID string) (*QueryConnectionResult, error) {
```

Comment:
GetConnection returns QueryConnectionResult. I feel it would make more readable if it returns Connection/ConnectionResult.

Reply:
agree.

Reply:
Rename to ConnectionResult